pkg/services/live/managedstream/runner.go

```diff
@@ -98,11 +98,22 @@ func (r *Runner) GetManagedChannels(orgID int64) ([]*ManagedChannel, error) {
 			Channel:    "plugin/testdata/random-labeled-stream",
 			Data:       frameJSON,
 			MinuteRate: 250,
-		}, &ManagedChannel{
-			Channel:    "plugin/testdata/random-20Hz-stream",
-			Data:       frameJSON,
-			MinuteRate: 1200,
 		})
+
+		var numberOf20hzChannels = 6
```

Comment:
@FZambia I am not sure this is the best way to do it. The goal is to have multiple streams of this type for tests, be it automated or manual. Would this be OK if I extracted `numberOf20hzChannels` to the grafana config? I would override it then in `custom.ini` with desired # before running tests

Reply:
I'd go with environment var for simplicity since users should not have access to this option yet.

Reply:
maybe better... 
* plugin/testdata/random-20Hz-stream
* plugin/testdata/random-20Hz-stream/a
* plugin/testdata/random-20Hz-stream/b
* plugin/testdata/random-20Hz-stream/c
* plugin/testdata/random-20Hz-stream/d

Do we need to define them all ahead of time?  or just create them as needed?